chalicelib/checks/wrangler_checks.py

```diff
@@ -1086,6 +1085,22 @@ def finalize_user_pending_labs(connection, **kwargs):
     return action
 
 
+def get_and_sort_tokens_to_string(s):
+    tokens = [t.lower() for t in re.split(r'\W+', s) if t]
+    tokens.sort()
+    return ''.join(tokens)
+
+
+def string_label_similarity(string1, string2):
```

Comment:
So this is telling me how similar two ordered series of strings are? Why are the tokens sorted? This seems quite well-described and it does what it does, but then when you use it below, it seems like it will do the wrong thing. Are you sure you want a function that _does_ sorting?